source/backend/api/Helpers/Middleware/LogRequestMiddleware.cs

```diff
@@ -19,6 +19,7 @@ public class LogRequestMiddleware
         private readonly RequestDelegate _next;
         private readonly ILogger<LogRequestMiddleware> _logger;
         private readonly RecyclableMemoryStreamManager _recyclableMemoryStreamManager;
+        private readonly int maxStreamLength = 2000;
```

Comment:
this number is arbitrary, but my thought is anything this large is of limited use in the log anyways.